### PR TITLE
Bootstrap multiple vulnerabilities

### PR DESCRIPTION
**Pull request by reynhout**
Let me know if these look good. Thanks!

### CODE REVIEW

File: gems/bootstrap-sass/CVE-2016-10735.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+gem: bootstrap-sass
+cve: 2016-10735
+url: https://blog.getbootstrap.com/2018/12/13/bootstrap-3-4-0/
+title: XSS vulnerability in bootstrap-sass
+date: 2016-07-27
+
+description: |
+  In Bootstrap 3.x before 3.4.0 and 4.x-beta before 4.0.0-beta.2,
```
Comment (reedloden) [inline gems/bootstrap-sass/CVE-2016-10735.yml]: Can you add an entry for `bootstrap` gem that covers this exact same issue but for the 4.x version(s)?

File: gems/bootstrap-sass/CVE-2016-10735.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+gem: bootstrap-sass
+cve: 2016-10735
+url: https://blog.getbootstrap.com/2018/12/13/bootstrap-3-4-0/
+title: XSS vulnerability in bootstrap-sass
```
Comment (reedloden) [inline gems/bootstrap-sass/CVE-2016-10735.yml]: Perhaps add "via data-target" or something so it's obvious to differentiate vulns?

File: gems/bootstrap/CVE-2018-14040.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+gem: bootstrap
+cve: 2018-14040
+url: https://blog.getbootstrap.com/2018/07/12/bootstrap-4-1-2/
+title: XSS vulnerabilities in bootstrap
```
Comment (reedloden) [inline gems/bootstrap/CVE-2018-14040.yml]: add "via data-parent, data-target, data-container" or via "data-*" properties or something? Just to help differentiate.

File: gems/bootstrap/CVE-2018-14040.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+date: 2018-07-03
+
+description: |
+  In Bootstrap before 4.1.2, XSS is possible in
```
Comment (reedloden) [inline gems/bootstrap/CVE-2018-14040.yml]: This is indented oddly... Extend to max 80 chars and re-indent?

File: gems/bootstrap-sass/CVE-2016-10735.yml (hunk 1 above)
Comment (reynhout) [reply gems/bootstrap-sass/CVE-2016-10735.yml]: Added as `gems/bootstrap/CVE-2016-10735.yml`

**Comment by reedloden** [comment]
Thank you so much!

**Pull request merged**